Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -67,27 +63,27 @@ public struct MessageView: View {
         }
     }
     
+    public enum Defaults {
+        static let hideMessagesWithRoles: Set<Chat.Role> = [.system, .function]
+    }
     
-    /// - Parameters:
-    ///   - chat: The chat message that should be displayed.
-    ///   - hideSystemMessages: If system messages should be hidden from the chat overview.
-    public init(_ chat: Chat, hideSystemMessages: Bool = true) {
+    public init(_ chat: Chat, hideMessagesWithRoles: Set<Chat.Role>) {
         self.chat = chat
-        self.hideSystemMessages = hideSystemMessages
+        self.hideMessagesWithRoles = MessageView.Defaults.hideMessagesWithRoles
```

Comment:
This current implementation would ignore everything that is passed in the initializer and just override with the default values:
```suggestion
        self.hideMessagesWithRoles = hideMessagesWithRoles
```